### PR TITLE
feat: more tsx tabs in RTL setup doc

### PR DESCRIPTION
**Pull request by Liadshiran**
As proposed in #696 and started with #700, I added the rest of the code snippets examples in Typescript for the Setup doc of react testing library.

### CODE REVIEW

**Comment by pedroapfilho** [comment]
Could you check these conflicts @Liadshiran

File: docs/react-testing-library/setup.mdx [hunk 1]
```diff
@@ -79,6 +79,7 @@ export { customRender as render }
 
 ```tsx title="test-utils.tsx"
 import React, { FC, ReactElement } from 'react'
+import { Omit } from 'react-redux'
```
Comment (MatanBobi) [inline docs/react-testing-library/setup.mdx:82]: I'm not sure about the `Omit` here.. I think that the options type should be different as it's the options for our `render` function.

**Review by MatanBobi — CHANGES_REQUESTED** [review]

**Comment by MatanBobi** [comment]
Thanks for taking the time @Liadshiran!
I left one comment :)

**Comment by MatanBobi** [comment]
And also, [here](https://github.com/testing-library/testing-library-docs/pull/828/files#diff-9a8513ec98fb23892fc832ebf723cc61ff3844a81e7b37f2a52d997426bb898cR100)
I think we should also omit `wrapper`..

**Comment by Liadshiran** [comment]
I added `wrapper` and removed `queries` which seemed redundant from the Omit type

**Review by MatanBobi — APPROVED** [review]

**Comment by MatanBobi** [comment]
Thanks @Liadshiran and @pedroapfilho! :)

**Pull request merged**